make fails with the following

```sh
docs git/master
❯ make content
node_modules/.bin/gulp content
[17:59:19] Using gulpfile ~/dev/wercker/docs/gulpfile.js
[17:59:19] Starting 'docs'...
[17:59:19] Finished 'docs' after 463 ms
[17:59:19] Starting 'modules'...
[17:59:19] Finished 'modules' after 748 μs
[17:59:19] Starting 'styles'...
[17:59:19] Finished 'styles' after 345 ms
[17:59:19] Starting 'content'...
[17:59:19] Finished 'content' after 10 μs

events.js:72
        throw er; // Unhandled 'error' event
              ^
SyntaxError: Unexpected token
make: *** [content] Error 8
```

--- a/docs-metrics/index.js
+++ b/docs-metrics/index.js
@@ -1,5 +1,5 @@
 // init keen
-window.client = new Keen({
+window.client = new window.Keen({
   projectId: "54d09b5359949a5f26b75fa4",
   writeKey:"25ca6a1eefdd4b3541ab7a6600153dc901a1eb0b6f2eba36a15a69acb2e12fad948e78312de7555a7f6a9ff718a0d129d2c3dd5223d09710f4fde015951a5e58e4f9c55e4d49d1bedfc973d4a2f4c82682a9c0ecf884bb841e400c47f58bb0f0677293f38d1192ff1852b37db9f1e052",
   readKey: "anything" // not used; not needed for data collection
@@ -36,7 +36,7 @@ newsletterJoin.onclick = trackCorporateClick;
 function trackSignupClick (e) {
   var buttonId  =  e.target.getAttribute('id');
   var pageId    =  document.querySelector('body').getAttribute('id');
-  return window.client.trackExternalLink(event, 'signup_click', {id: buttonId, page: pageId});
+  return window.client.trackExternalLink(e, 'signup_click', {id: buttonId, page: pageId});
 }
 
 /**
@@ -45,5 +45,5 @@ function trackSignupClick (e) {
 function trackCorporateClick (e) {
   var buttonId  =  e.target.getAttribute('id');
   var pageId    =  document.querySelector('body').getAttribute('id');
-  return window.client.trackExternalLink(event, "corporate_click", {id: buttonId, page: pageId});
+  return window.client.trackExternalLink(e, "corporate_click", {id: buttonId, page: pageId});
 }

--- a/docs-sidebar/toc-docs.json
+++ b/docs-sidebar/toc-docs.json
@@ -97,7 +97,7 @@
   ],
   [
      "git",
-     "pull-requests.md"
+     "pull-requests.md",
      "submodules.md"
   ],
   [

